docs/README_EN.md

```diff
@@ -390,4 +390,4 @@ This project is licensed under the [MIT LICENSE](/LICENSE).
 
 This project is inspired by great open source projects designed for beginners, such as [first-contributions](https://github.com/firstcontributions/first-contributions) and [Contribute-To-This-Project](https://github.com/Syknapse/Contribute-To-This-Project).
 
-[twttr-share]: https://twitter.com/intent/tweet?text=Start your OSS activities with a tutorial in full Japanese🚀&url=https://github.com/first-contributions-ja/first-contributions-ja.github.io&hashtags=Programming 'Tweet this project'
\ No newline at end of file
+[twttr-share]: https://twitter.com/intent/tweet?text=Start your OSS activities with a tutorial in full Japanese🚀&url=https://github.com/first-contributions-ja/first-contributions-ja.github.io&hashtags=Programming 'Tweet this project'
```

Comment:
Spaces in 
`Start your OSS activities with a tutorial in full Japanese🚀`
should be replaced with `%20`.

This allows it to be treated as a single URL string.